mdoc/src/main/scala/mdoc/internal/worksheets/EvaluatedWorksheet.scala

```diff
@@ -11,5 +11,6 @@ case class EvaluatedWorksheet(
     val scalac: ju.List[String],
     val classpath: ju.List[Path],
     val dependencies: ju.List[coursierapi.Dependency],
-    val repositories: ju.List[coursierapi.Repository]
+    val repositories: ju.List[coursierapi.Repository],
+    val exportableEvaluation: ju.Optional[String]
```

Comment:
I think it would be nice if we can export the evaluation on-demand. Most of the time I suspect this rendering won't be used. Evaluating worksheets is already quite expensive (compile, classload, potentially slow code, ...).